nextroute/schema/input.go

```diff
@@ -10,14 +10,14 @@ import (
 // Input is the default input schema for nextroute.
 type Input struct {
 	// Options contains arbitrary options specified by the user.
-	Options        *any                 `json:"options,omitempty"`
-	Defaults       *Defaults            `json:"defaults,omitempty"`
-	Vehicles       []Vehicle            `json:"vehicles,omitempty"`
-	Stops          []Stop               `json:"stops,omitempty"`
-	StopGroups     [][]string           `json:"stop_groups,omitempty"`
-	AlternateStops []Stop               `json:"alternate_stops,omitempty"`
-	DurationGroups []route.ServiceGroup `json:"duration_groups,omitempty"`
-	DurationMatrix [][]float64          `json:"duration_matrix,omitempty"`
+	Options        *any                  `json:"options,omitempty"`
```

Comment:
Do we actually need `Options` and if  we do, shouldn't this be of type `NextRouteOptions` (currently still found in `platform/nextapp/main.go`)?

Reply:
other than that: a pointer to an interface is not really needed but probably does not hurt either

Reply:
Completely right on the pointer to interface thing, my bad. Other than that, @larsbeck what should we do about these options and of what type should they be?